### PR TITLE
pass-task

### PR DESCRIPTION
**Pull request by TakenPilot**
Patch
- Report what was rendered and how long it took using `hrtime`.
- Pass composer-level route with `next()` if NotFoundError

### CODE REVIEW

File: lib/composer.js [hunk 1]
```diff
@@ -365,19 +365,28 @@ function getExpressRoutePrefix(site) {
...
-      res.send(html);
-    }).catch(function (err) {
+  renderUri(pageReference, res).then(function (html) {
+    res.type('html');
+    res.send(html);
+
+    log.info('rendered express route:' +
+      (req.hostname + req.baseUrl + req.path) + ' ' +
+      ((process.hrtime()[1] - hrStart) / 1000000) + 'ms'
+    );
```
Comment (cruzanmo) [inline lib/composer.js]: Should it always log or only when there is an error?

Comment (TakenPilot) [inline lib/composer.js]: Always.  Seems we need a bigger way to hit people when they're putting logic in the GET.

Comment (TakenPilot) [inline lib/composer.js]: Right now, we're sitting at 100ms on local.  :( :( :(

**Comment by cruzanmo** [comment]
+1

**Comment by yoshokatana** [comment]
:+1:

**Pull request merged**